src/Common/src/Interop/Windows/Crypt32/Interop.certificates_imports.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I'm not opposed to the types file being separate, but the _imports suffix/infix should be removed from this one (and any other _imports.cs files).